README.md

```diff
@@ -83,6 +85,18 @@ export const config = {
 
 This exposes MCP endpoints at `/api/tidewave/mcp` and `/api/tidewave/shell`.
 
+Then create `middleware.ts` with:
+
+```typescript
+import { toNextMiddleware } from 'tidewave/next-js';
```

Comment:
Suggestion: let's rename it to `tidewaveMiddleware`, so looking at the code it is clear it is a tidewave thingy.